<ui version="4.0" >
 <class>PreferencesDialogClass</class>
 <widget class="QDialog" name="PreferencesDialogClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="fontsTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>335</width>
        <height>188</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Fonts</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="fontLabel" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Preferred" hsizetype="Maximum" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text" >
            <string>Font settings:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox" >
           <item>
            <property name="text" >
             <string>Browser</string>
            </property>
           </item>
           <item>
            <property name="text" >
             <string>Application</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0" >
        <widget class="QStackedWidget" name="stackedWidget_2" >
         <property name="currentIndex" >
          <number>0</number>
         </property>
         <widget class="QWidget" name="page_4" >
          <property name="geometry" >
           <rect>
            <x>0</x>
            <y>0</y>
            <width>317</width>
            <height>142</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="filtersTab" >
      <property name="geometry" >
       <rect>
        <x>8</x>
        <y>60</y>
        <width>335</width>
        <height>188</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Filters</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" >
        <widget class="QLabel" name="label_2" >
         <property name="frameShape" >
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="text" >
          <string>Attributes:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2" >
        <widget class="QListWidget" name="filterWidget" />
       </item>
       <item rowspan="2" row="1" column="2" >
        <widget class="QTreeWidget" name="attributeWidget" >
         <column>
          <property name="text" >
           <string>1</string>
          </property>
         </column>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="filterAddButton" >
         <property name="text" >
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="filterRemoveButton" >
         <property name="text" >
          <string>Remove</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="docsTab" >
      <property name="geometry" >
       <rect>
        <x>8</x>
        <y>60</y>
        <width>335</width>
        <height>188</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Documentation</string>
      </attribute>
      <layout class="QVBoxLayout" >
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Registered Documentation:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="margin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QListWidget" name="registeredDocsListWidget" />
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="docAddButton" >
             <property name="text" >
              <string>Add...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="docRemoveButton" >
             <property name="text" >
              <string>Remove</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer>
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="networkTab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>335</width>
        <height>188</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>Network</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout" >
       <item>
        <widget class="QGroupBox" name="proxyBox" >
         <property name="title" >
          <string>Use Http Proxy</string>
         </property>
         <property name="checkable" >
          <bool>true</bool>
         </property>
         <property name="checked" >
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout" >
          <item row="0" column="0" >
           <widget class="QLabel" name="label_4" >
            <property name="text" >
             <string>Http Proxy:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" >
           <widget class="QLineEdit" name="proxyLineEdit" />
          </item>
          <item row="1" column="0" >
           <widget class="QLabel" name="label_5" >
            <property name="text" >
             <string>Port:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" >
           <widget class="QSpinBox" name="portSpinBox" >
            <property name="maximum" >
             <number>65535</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>81</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons" >
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget_2</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>375</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel" >
     <x>347</x>
     <y>125</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
